.gitignore

```diff
@@ -1,3 +1,5 @@
 *.pyc
-/config.py
+/coloringbook/config.py
```

Comment:
The caps in `CONFIG.py` were based on my misreading of the README, which used all-caps "CONFIG" as a placeholder for 'the actual name of the config file'. There is no reason why the name of the configuration file should be in all caps.